### PR TITLE
Create form

### PR DESCRIPTION
**Pull request by AnnaKanska**
Created a bootstrap form
Created this.state properties
Created updateInput method
Altered home view to access other views

### CODE REVIEW

**Comment by NyanHorse** [comment]
what did you do on database-access.js?

File: src/components/createView.jsx [hunk 1]
```diff
@@ -1,16 +1,67 @@
...
+
+  updateInput (e) {
+    let name = e.target.name;
+    this.setState({
+      [name]: e.target.value
+    });
   }
 
   render () {
+    console.log(this.state.name, this.state.placeDescription, this.setState.lng);
```
Comment (kmoroder) [inline src/components/createView.jsx]: Remove console.logs before making pull requests!

File: src/components/createView.jsx [hunk 2]
```diff
@@ -1,16 +1,67 @@
...
+            <InputGroup>
+              <FormGroup className="col-md-4">
+                <h3>Place 1</h3>
+                <FormControl name="placeName" className="col-md-4" type="text" required onChange={e => this.updateInput(e)}></FormControl>
+                <h3>Describe the task</h3>
+                <FormControl name="placeDescription" componentClass="textarea" placeholder="In this place, you need to do ... and find ..." required onChange={e => this.updateInput(e)}/>
+              </FormGroup>
+              <FormGroup>
+                <h3>Insert geolocation of this place!</h3>
+                {/* Need to fix inline */}
```
Comment (kmoroder) [inline src/components/createView.jsx]: What does "need to fix inline" mean? Remove this comment from your code!

File: src/services/common/database-access.js [hunk 3]
```diff
@@ -1,32 +1,36 @@
-const pg = require('pg');
-const connectionString = process.env.DATABASE_URL || 'postgres://localhost:5432/todos';
+const pg = require("pg");
```
Comment (kmoroder) [inline src/services/common/database-access.js]: Please remove the changes in this file. It was not necessary to reformat and will cause merge conflicts with Almudena's changes.

**Pull request merged**